howto/stream/exchange-oob-data.md

```diff
@@ -314,15 +366,23 @@ amc launch --raw jammy:android12:amd64 \
 Enabling the `allow_custom_system_signatures` feature is required to run the Android application as a system app in an Android container.
 ```
 
-Once the session is active:
+In the [Anbox Streaming SDK](https://github.com/canonical/anbox-streaming-sdk), there is an [out_of_band_v2](https://github.com/canonical/anbox-streaming-sdk/tree/master/examples/android/out_of_band_v2) project. You can either:
+- compile and modify the example application to meet your needs.
+- use the prebuilt out-of-band v2 APK from the [release tarball](https://github.com/canonical/anbox-streaming-sdk/releases) for a quicker test of the functionality.
 
-1. Use the web client, which creates a custom data channel `foo` to initiate the WebRTC connection.
-2. After the WebRTC connection is established, verify that the Android application receives a notification indicating the availability of the `foo` data channel.
-3. Confirm that the Android application can connect to the `foo` data channel created by the web client.
-4. Send messages from either the Android application or the WebRTC client, and ensure that the messages are successfully sent and and received over the `foo` data channel.
+The following instructions use an APK built from this example for end-to-end testing. To do so:
+  1. Use the out-of-band v2 APK in the above addon creation.
+  1. Launch a stream-enabled instance with the addon created as mentioned above.
+  1. Retrieve the session id that asssoicated to the `test-oobv2` instance
+
+         amc ls --filter name=test-oobv2 --format=csv | awk -F',' '{split($6, r, "="); print r[2]}'
+
+  1. To join the session, make sure to replace `<session_id>` the session ID retrieved above in the stream client.
+  1. Launch the stream client that extends to [create the `foo` data channel](https://documentation.ubuntu.com/anbox-cloud/en/latest/howto/stream/exchange-oob-data/#prepare-your-web-application) upon establishing a WebRTC connection.
+  1. Once the WebRTC connection is connect, open the Out of Band v2 application in the Android container. enter 'foo' as the channel name in the line edit widget, then click the `CONNECT` button. Check if a toast message saying 'Channel "foo" is connected' appears.
```

Comment:
```suggestion
  1. Once the WebRTC connection is successful, open the Out of Band v2 application in the Android container. enter 'foo' as the channel name in the line edit widget, then click *CONNECT*. Check if a toast message saying 'Channel "foo" is connected' appears.
```

Reply:
Maybe 
Once the WebRTC connection is connect,
->
Once the WebRTC connection is established